### DISCUSSION TITLE
snake-game code review IM-13

### DISCUSSION THREAD

**Discussion by ImLewel** [HowProgrammingWorks/LiveQA#45] [Code review]
## Old repo https://github.com/ImLewel/SnakeGame

**Comment by tshemsedinov** [comment]
See first review: https://github.com/ImLewel/SnakeGame/issues/3

**Reply by ImLewel** [reply]
Here is my **new** [**repo**](https://github.com/ImLewel/snake-game) (https://github.com/ImLewel/snake-game) (issues are transfered).

This [**link**](https://github.com/ImLewel/SnakeGame) (https://github.com/ImLewel/SnakeGame) **is an old one**

I was having issues with commits, the local repository wasn't working properly, there was a lot of junk, so I decided to create a new repository and make things a lot better. I didn't lose many commits, especially important ones. The more important commits are already on the new repository, sorry for the inconvenience.

**Reply by ImLewel** [reply]
Waiting for review 2
